content/en/profiler/enabling/java.md

```diff
@@ -236,9 +236,12 @@ The wallclock engine does not depend on the `/proc/sys/kernel/perf_event_paranoi
 {{< tabs >}}
 {{% tab "JFR" %}}
 The JFR based allocation profiling engine is enabled by default since JDK 16.
-The reason it's not enabled by default for JDK 8 and 11, is that an allocation intensive
-application can lead to high overhead and large recording sizes.
-To enable it for JDK 8 and 11, add the following:
+The reason it is not enabled by default for Open JDK 8 and JDK 11, is that an allocation intensive
+application can lead to high overhead and large recording sizes. 
+This is not supported for Oracle JDK 8 and IBM J9 JDK 8. A paid license version of Oracle JDK 8
+enables the usage of JFR events on which this feature relies.
```

Comment:
> A paid license version of Oracle JDK 8
> enables the usage of JFR events on which this feature relies.

I'm actually not sure if this is the case. May be worth checking.

Reply:
The usage of the `UnlockCommercialFeatures` seems to be required, though identifying what versions need this seems difficult. I do not think we want to go into this level of detail.